vue/components/ui/organisms/header/header.vue

```diff
@@ -337,6 +402,12 @@ export const Header = {
                 });
             }
             return items;
+        },
+        announcementsToRead() {
+            const current = Date.now();
+            return this.announcements.items.length === 0
+                ? false
+                : this.announcements.items[0].timestamp > current - this.announcements.new_delta;

```

Comment:
Sort them first by timestamp, not doing so is sort of a big assumption.

Reply:
fixed in header-pulse of ripe-pulse